<launch>
    <!-- Launch Gazebo with the maze world -->
    <include file="$(find mir_gazebo)/launch/mir_maze_world.launch" />

    <!-- MiR 100 in Gazebo -->
    <arg name="mir_type" default="mir_100" />
    
     <!-- Publish the initial pose -->
    <node pkg="tf2_ros" type="static_transform_publisher" name="initial_pose_publisher" args="10.0 10.0 0 0 0 0 1 map odom" />

    <!-- RViz Visualization -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find mir_navigation)/rviz/navigation.rviz" />
  

</launch>
